system/include/emscripten/xmmintrin.h

```diff
@@ -483,6 +508,9 @@ _mm_cmpnlt_ss(__m128 __a, __m128 __b)
 static __inline__ int __attribute__((__always_inline__))
 _mm_comieq_ss(__m128 __a, __m128 __b)
 {
+#ifdef __EMSCRIPTEN__
+  if (isnanf(__a[0])) return 1;
+#endif
```

Comment:
Thanks, the asymmetric handling was an oversight. Fixed that and added a test at https://github.com/juj/emscripten/commit/d83d875c11f2ad7f2955dcb117fe653131eb87bf .